gef-ui/server/controllers/supporting-information/security-details/index.js

```diff
@@ -109,9 +109,6 @@ const postSecurityDetails = async (req, res) => {
 
     await updateApplication(dealId, application);
 
-    if (application.supportingInformation?.requiredFields?.includes('exportLicence')) {
```

Comment:
does the security-details index.test.js need updating?

Reply:
it didn't have a test before. so removing it doesn't do anything to that test